adagrams/game.py

```diff
@@ -1,13 +1,80 @@
 from random import randint
-
+letters_pool={"A":9,"B":2,"C":2,"D":4,"E":12,"F":2,"G":3,"H":2,"I":9,"J":1,"K":1,"L":4,"M":2,"N":6,"O":8,"P":2,"Q":1,"R":6,"S":4,"T":6,"U":4,"V":2,"W":2,"X":1,"Y":2,"Z":1}
 def draw_letters():
-    pass
+    drawed_letters=[]
+    all_letters=[]
+    letters_pool={"A":9,"B":2,"C":2,"D":4,"E":12,"F":2,"G":3,"H":2,"I":9,"J":1,"K":1,"L":4,"M":2,"N":6,"O":8,"P":2,"Q":1,"R":6,"S":4,"T":6,"U":4,"V":2,"W":2,"X":1,"Y":2,"Z":1}
+    for letter, quantity in letters_pool.items():
+        for i in range(quantity):
+            all_letters.append(letter)
+    while len(drawed_letters)<10:
+        random_number= randint(0,len(all_letters)-1) 
```

Comment:
Prefer this variable to be a little more descriptive, while the variable does reference a random number, what __is__ that random number? The random number is a random index so a name like `rand_int` would be more descriptive and make your code more self-documenting